.github/workflows/basic-tests.yml

```diff
@@ -26,23 +26,34 @@ jobs:
         sudo apt-get -o Acquire::Retries=3 update
         sudo apt-get -o Acquire::Retries=3 install *fftw3* mpi intel-mkl* graphviz
         pip install tox pip setuptools --upgrade
-    - name: installing auxiliary data files
+    - name: Cache LAL auxiliary data files
```

Comment:
Would you be able to add a link to the instructions on how to do this into this PR? Just so we have it on-hand in case something breaks in the future

Reply:
It looks like https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/caching-dependencies-to-speed-up-workflows, but if you used something else, do say

Reply:
Yes that is what I used, though I also looked at what Duncan did in `mac-test.yml`.